matsim/src/main/java/org/matsim/core/controler/corelisteners/PlansDumpingImpl.java

```diff
@@ -66,6 +66,11 @@ final class PlansDumpingImpl implements PlansDumping, BeforeMobsimListener {
 		this.writePlansInterval = config.getWritePlansInterval();
 		this.writeMoreUntilIteration = config.getWritePlansUntilIteration() ;
 	}
+	
+	@Override
+	public double priority() { // this should happen after other BeforeMobsimListeners did their thing
+		return -1000;
```

Comment:
You should make this a public final static constant, so other listeners can make use of it to control if they want to run before or after the plans dumping.